### DISCUSSION TITLE
Search Mode Post Screening

### DISCUSSION THREAD

**Discussion by GregoryDams** [asreview/asreview#1031] [Ideas]
## Feature Request
Good day,
First, thank you for providing ASReview open-source to the academic community, this is an incredible development sure to help many!

One problem I have encountered was that once I have selected a small range of relevant and irrelevant articles, that the subsequent review process was (understandably) heavily weighted on the initial articles. For example, I'm looking through several mental health studies and after marking perhaps too many articles focused on depression as relevant, I now seem to only get depression articles to screen. 

If it was possible, it would be very helpful to allow for manual control in the screening process to allow the search feature or random study review again so that meaningfully different studies can be selected. The alternative I have is to save my current results (and selections) and restart the screening procedure to manually search for a mix of the articles I previously marked relevant as well as studies that can seed greater variety.

**Comment by GerbrichFerdinands** [comment]
Hi @GregoryDams, thank you! 😁 

At the moment the feature of including additional 'key papers' during the screening process is not supported in ASReview. 

I see a couple of solutions: 

In ASReview, you export your decisions. You upload this excel or csv file in a new ASReview project.
- In the new project you can select extra key papers at the 'Prior Knowledge' step. Your previous decisions will carry over to this new project (see [this related question in the FAQ](https://asreview.readthedocs.io/en/latest/intro/faq.html#related-question-how-can-i-add-more-publications-while-i-are-already-started-screening-in-asreview))
- At the "Select Active learning model" step, you can pick the 'Mixed' query strategy instead of the default setting 'Certainty-based'. Then, instead of always seeing the most relevant study (default setting), you see the most relevant study x% of the time and a randomly selected study y% of the time. (@J535D165 @terrymyc can you help me fill in the x/y? I'm not sure!!)

![image](https://user-images.githubusercontent.com/45358236/110088892-baa66c00-7d95-11eb-99aa-2148936f1259.png)

You can even apply the 2 strategies mentioned above at the same time.

Also, a question you might want to ask yourself is "what kind of articles am I looking to include"? If it really are different topics you want to look for in the same dataset - for example, papers on anxiety and depression - it might even be best to go through the dataset twice. 

Many thanks for filing this issue, there must be other users that also encounter this problem!
